code/switch.js

```diff
@@ -0,0 +1,227 @@
+                console.log(order);
+                alert("you chose " + order);
+                ageCheck();
+                break;
+
+            case 2:
+                orderType.push(menu[1]);
+
+                ordertemp.push(menu[1].subtype);
+                ordertemp.push(" ");
+                ordertemp.push(menu[1].type);
+                orderjoin = (ordertemp.join(""));
+                order.push(orderjoin);
+                alert("you chose " + order);
+                ageCheck();
+                break;
+
+            case 3:
+                orderType.push(menu[2]);
+                ordertemp.push(menu[2].subtype);
+                ordertemp.push(" ");
+                ordertemp.push(menu[2].type);
+                orderjoin = (ordertemp.join(""));
+                order.push(orderjoin);
+                alert("you chose " + order);
+                ageCheck();
+                break;
+
+            default:
+                alert("That's not on the menu");
+                break;
+        }
+        break;
+    case 2:
+        let pastaOption = parseInt(prompt("Press 1 for 🍝 " + menu[3].subtype + " , \n press 2 for 🥟 " + menu[4].subtype + " \n press 3 for 🍜 " + menu[5].subtype));
+        switch (pastaOption) {
+            case 1:
+                orderType.push(menu[3])
+                order.push(menu[3].subtype)
+                alert("you chose " + order);
+                ageCheck();
+                break;
+            case 2:
+                orderType.push(menu[4])
+                order.push(menu[4].subtype)
+                alert("you chose " + order);
+                ageCheck();
+                break;
+            case 3:
+                orderType.push(menu[5])
+                order.push(menu[5].subtype)
+                alert("you chose " + order);
+                ageCheck();
+                break;
+            default:
+                alert("Sorry that's not on the  menu")
+                break;
+        }
+        break;
+
+    case 3:
+        let saladOption = parseInt(prompt("Press 1  🏛️🥬 for " + menu[6].subtype + " \n press 2 for 🇬🇷🥬 " + menu[7].subtype + " \n  press 3 for 🍉🥬  " + menu[8].subtype));
+        switch (saladOption) {
+            case 1:
+                orderType.push(menu[6])
+                order.push(menu[6].subtype)
+                alert("you chose " + order);
+                ageCheck();
+                break;
+            case 2:
+                orderType.push(menu[7])
+                order.push(menu[7].subtype)
+                alert("you chose " + order);
+                ageCheck();
+                break;
+            case 3:
+                orderType.push(menu[8])
+                order.push(menu[8].subtype)
+                alert("you chose " + order);
+                ageCheck();
+                break;
+            default:
+                alert("Sorry that's not on the  menu")
+                break;
+        }
+        break;
+}
```

Comment:
Make sure to use a formatter to have the correct spacing and indentation, and change the tab size settings from 4 to 2😄 you can see now there is a lot of empty spaces in the code that compromise the readability ✨